design.md

```diff
@@ -2,12 +2,25 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+I'm assuming the user will either be familiar with Picobot or won't (so, everyone lol), but it is designed with natural language in mind to be easily understood and accessible to users.
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
+I choose this design to give flexibility to users of the DSL. Despite being nearly done with the Mudd CS major, I completely forgot Picobot and had to relearn it. Based on this, I changed the language to be both easily understood as well as give the user flexibility if they want to use syntax from previous Picobot and this new syntax.
+
 ## What behaviors are easier to express in your design than in Picobot’s original design?  If there are no such behaviors, why not?
 
+I think the one that stands out is jumping from the empty state back into the same state. By default, not listing a state to go to will go back to the state the picobot is already in, which should result in less code being wrote.
```

Comment:
Definitely agree with this, I really like how you cut this out if the user didn't need to change states